packages/discovery-provider/integration_tests/tasks/test_get_feed_es.py

```diff
@@ -100,20 +138,30 @@ def test_get_feed_es(app):
     # test feed
     with app.app_context():
         feed_results = get_feed_es({"user_id": "1"})
-        assert len(feed_results) == 2
+        print(f"len feed_results: {len(feed_results)}")
+        print(f"type: {[z.get('track_id') for z in feed_results]}")
+        # print(f"type: {[z.get('playlist_id', z.get('track_id')) for z in feed_results]}")
+        print(f"feed_results: {feed_results}")
+        assert len(feed_results) == 4
         assert feed_results[0]["playlist_id"] == 1
         assert feed_results[0]["save_count"] == 1
         assert len(feed_results[0]["followee_reposts"]) == 1
 
         assert feed_results[1]["track_id"] == 1
         assert feed_results[1]["save_count"] == 1
 
+        assert feed_results[2]["playlist_id"] == 3
+        assert feed_results[3]["track_id"] == 5
+
         # Test offset works
         feed_results = get_feed_es({"user_id": "1"}, offset=1)
-        assert len(feed_results) == 1
+        assert len(feed_results) == 3
         assert feed_results[0]["track_id"] == 1
         assert feed_results[0]["save_count"] == 1
 
+        assert feed_results[1]["playlist_id"] == 3
+        assert feed_results[2]["track_id"] == 5
+
```

Comment:
is there any sense in iterating through `feed_results` and asserting `playlist_id: 4` is not present?

Reply:
no because there is an assertion for the length of the results and the ids at each index